CodeSniffer/Standards/Generic/Sniffs/PHP/SyntaxSniff.php

```diff
@@ -56,7 +56,7 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
     {
         $phpPath = PHP_CodeSniffer::getConfigData('php_path');
         if ($phpPath === null) {
-            return;
+            $phpPath = PHP_BINARY;
```

Comment:
I've added fallbacks using PHP_BINDIR, looking for php and php.exe. Any other commonly distributed php executable names? I figure anything unique can still be configured using php_path.